### PR TITLE
Clean CLI docs

### PR DESCRIPTION
**Pull request by soltysh**
This PR includes following fixes:
- explains loglevels used in CLI
- removes unused entries
- fixes misconceptions

This should be included in 3.11.

@juanvallejo @ahardin-rh ptal

### CODE REVIEW

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -76,65 +76,43 @@ No events.
...
-pass `--api-version` in order to point the `oc` client to the correct API
-endpoint. For example: `oc get svc --api-version=v1`.
-====
-
 [[object-types]]
 
 == Object Types
-The CLI supports the following object types, some of which have abbreviated
-syntax:
+Below is the list of the most common object types CLI supports, some of which
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: the CLI supports

File: cli_reference/basic_cli_operations.adoc [hunk 2]
```diff
@@ -76,65 +76,43 @@ No events.
...
+|`Service` |`svc`
 |`ServiceAccount` | `sa`
-|`persistentVolume` |`pv`
-|`persistentVolumeClaim` |`pvc`
+|`StatefulSets` | `sts`
+|`PersistentVolume` |`pv`
+|`PersistentVolumeClaim` |`pvc`
 |===
 
+If you want to know the full list of resources the server supports use `oc api-resources`.
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: supports, use

File: cli_reference/basic_cli_operations.adoc [hunk 3]
```diff
@@ -754,3 +717,22 @@ link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
 the command is executed by a `cluster-admin` user. Administrators can SSH into
 a node host, then use the `docker exec` command on the desired container.
 ====
+
+[[troubleshooting-cli]]
+
+== Troubleshooting oc
+At any point in time you can get more verbosed output from any command by increasing
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: s/At any point in time you can get more verbosed output/You can get more verbosed output

File: cli_reference/basic_cli_operations.adoc [hunk 4]
```diff
@@ -754,3 +717,22 @@ link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
...
+the loglevel using `-v=X` flag. By default, the loglevel is set to 0, but you can
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: set to  `0`

    from `0` to `10`

File: cli_reference/basic_cli_operations.adoc [hunk 5]
```diff
@@ -754,3 +717,22 @@ link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
...
+set its value from 0 to 10. Below you shall find explanation what you might expect
+at each loglevel.
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: I would remove this last sentence and replace it with this just before the list:

.Overview of each loglevel

that should render as an example heading

**Comment by ahardin-rh** [comment]
@soltysh Thank you for this! Just a few comments from me. :beers:

**Comment by soltysh** [comment]
@ahardin-rh update, ptal

**Comment by ahardin-rh** [comment]
LGTM! Thank you!

**Pull request merged**